File: src/main/java/co/interleap/mocks/EmailBody.java
package co.interleap.mocks;

public class EmailBody {
    public final String subject;
    public final String body;
    public final String receiver;

    public EmailBody(String subject, String body, String receiver) {
        this.subject = subject;
        this.body = body;
        this.receiver = receiver;
    }
}

File: src/main/java/co/interleap/mocks/EmailBody.kt
package co.interleap.mocks

class EmailBody(val subject: String, val body: String, val receiver: String){

    // When the test is not passing due to has code issue then you have to implement equals
    override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if(other == null || javaClass != other.javaClass) return false
        return true
    }
}